mgmtsystem_audit/views/res_users.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
```

Comment:
could you remove the Copyright from the xml?
Its uncommon to copyright view definitions.